.github/workflows/iroha2-release-pr.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         with:
           context: .
           push: false
+          load: true
           tags: hyperledger/iroha2:dev
```

Comment:
Making the locally built `hyperledger/iroha2:dev` image available for subsequent steps